### DISCUSSION TITLE
Revit Lookup 2025 ?

### DISCUSSION THREAD

**Discussion by LoRue656Plus10** [jeremytammik/RevitLookup#203] [General]
I was curious if there has been any discussion or planning for a Revit Lookup 2025?  You are probably aware they are migrating Revit to .NET 6 (or 8?).  I know the pyRevit team is looking into this already.  Any info is appreciated.  I use Lookup daily and it has been incredibly helpful.

Best,
@LoRue656Plus10

**Comment by Nice3point** [comment]
I will update RevitLookup as soon as a new version of Revit is released, it shouldn't take more than half an hour

**Reply by LoRue656Plus10** [reply]
Wow, I did not see that coming!  Really?  I guess those types of add-ins are not so much affected by the change.
Thanks for clearing that up.

Best,
@LoRue656Plus10

**Reply by Nice3point** [reply]
@LoRue656Plus10 RevitLookup was originally designed for use on .Net 5+, no ".Net Framework only" dependent libraries are used there. The code is already adapted, and all I have to do is to upgrade TargetFramework to net8 and test it
